### PR TITLE
Map 1960: Adding change used for link on a cell level

### PR DESCRIPTION
**Pull request by ashleygyngell**
- [MAP-1960](https://dsdmoj.atlassian.net/jira/software/c/projects/MAP/boards/1354?assignee=6419ad1b67102fc717c0d25e&selectedIssue=MAP-1960): Allows user, at residential cell level, to make changes to the used for when making making changes to a location.

- Small update to readme for running int-ui tests 

[MAP-1960]: https://dsdmoj.atlassian.net/browse/MAP-1960?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ so that assets are rebuilt when modified and tests will re-run:
 
 ```shell
 docker compose -f docker-compose-test.yml up
-npm run start-feature:dev
+npm run start-feature
```
Comment (marcusaleman) [inline README.md:98]: After discussion this has worked for me as well.
However I think we should agree about this change. -> i will do LGTM for now.

File: integration_tests/e2e/changeUsedFor/index.cy.ts [hunk 2]
```diff
@@ -113,7 +112,29 @@ context('Set cell type', () => {
         })
       })
 
-      it('shows the correct checked checkbox list', () => {
+      it('does show a checked checkbox list on a cell level when multiple are checked', () => {
+        cy.task('stubLocations', locationAsCell)
```
Comment (marcusaleman) [inline integration_tests/e2e/changeUsedFor/index.cy.ts:116]: ok

**Review by marcusaleman — APPROVED** [review]
good stuff
LGTM

**Review by Mjwillis — APPROVED** [review]
LGTM

**Pull request merged**